server/app-engine/gtfsrt.py

```diff
@@ -176,12 +176,30 @@ def make_trip_update(list):
 
     return entity
 
-def alert_is_current(alert):
+def alert_is_current(alert, service_alert_ui):
```

Comment:
`service_alert_ui` is a pretty terrible name for this arg. I feel like it was called `include_future_alerts` before, and that's a much better name. There is no need for the server to know anything about a service alert UI.

Reply:
The other option would be to have two new boolean parameters for get_alert_feed():
1. use_cache
2. include_future_alerts
I had consolidated them into "service_alert_ui", and I see what you mean about that not being relevant to the API. I'll update.